### ISSUE TITLE
Functionally Equivalent Extraction delta converges to 0

### ISSUE THREAD

**Issue by Dainiuss1** [Trusted-AI/adversarial-robustness-toolbox#867]
Summary:
Functionally Equivalent Extraction can't find critical points, because delta converges to 0 and that causes infinite loop.

Currently facing same problem as @puro2xz and @xiaoxianghuayu in https://github.com/Trusted-AI/adversarial-robustness-toolbox/issues/420 when delta converges to 0 and causes infinite loop. Already tried to use np.random.seed(905), np.random.seed(1), however these changes didn't fix my problem.

Maybe have any thoughts how can I solve this situation?

 - OS (Ubuntu 20.04.1)
 - Python version  (3.7)
 - ART version or commit number (1.5.1)
 - TensorFlow (1.15.2)/ Keras (2.3.1) / PyTorch / MXNet (1.6.0)
 
Thank you in advance!

**Comment by beat-buesser** [comment]
Hi @Dainiuss1 Thank you very much for using ART! Have you been able to identify the reason why delta converges to 0 in your case?

**Comment by Dainiuss1** [comment]
Hi, @beat-buesser 

Sadly, but no. I tried different numpy, tensorflow, keras versions, also tried to find seed that would solved the issue, however nothing helped me

**Comment by beat-buesser** [comment]
Have you been able to visualise the critical points of you model? Can you post a graph of logits as function of `t_current`?
